scripts/o3de/o3de/utils.py

```diff
@@ -134,20 +135,25 @@ def download_file(parsed_uri, download_path: pathlib.Path, force_overwrite: bool
                 return 1
 
     if parsed_uri.scheme in ['http', 'https', 'ftp', 'ftps']:
-        with urllib.request.urlopen(parsed_uri.geturl()) as s:
-            download_file_size = 0
-            try:
-                download_file_size = s.headers['content-length']
-            except KeyError:
-                pass
-            def download_progress(downloaded_bytes):
-                if download_progress_callback:
-                    return download_progress_callback(int(downloaded_bytes), int(download_file_size))
-                return False
-            with download_path.open('wb') as f:
-                download_cancelled = copyfileobj(s, f, download_progress)
-                if download_cancelled:
-                    return 1
+        try:
+            with urllib.request.urlopen(parsed_uri.geturl()) as s:
+                download_file_size = 0
+                try:
+                    download_file_size = s.headers['content-length']
+                except KeyError:
+                    pass
+                def download_progress(downloaded_bytes):
+                    if download_progress_callback:
+                        return download_progress_callback(int(downloaded_bytes), int(download_file_size))
+                    return False
+                with download_path.open('wb') as f:
+                    download_cancelled = copyfileobj(s, f, download_progress)
+                    if download_cancelled:
+                        logger.warn(f'Download of file to {download_path} cancelled.')
```

Comment:
Seems more like info the warn

Reply:
I agree.
Also logger.warn is [deprecated](https://docs.python.org/3/library/logging.html#logging.warning) and python likes to "warn" about it